sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -238,6 +257,28 @@ make_kernel_bundle<backend::ext_oneapi_level_zero,
     const context &Context)
 ```
 </td>
+<td>Constructs a SYCL kernel instance from a Level-Zero
+<code>ze_kernel_handle_t</code>. The <code>KernelBundle</code> input structure
+specifies the <code>kernel_bundle<bundle_state::executable></code> corresponding
+to the Level-Zero module from which the kernel is created. There must be exactly
+one Level-Zero module in the <code>KernelBundle</code>. The <code>Context</code>
+argument must be a valid SYCL context encapsulating a Level-Zero context, and
+the Level-Zero module must be created on the same context.
+The <code>Ownership</code> input structure member specifies if the SYCL runtime
+should take ownership of the passed native handle. The default behavior is to
+transfer the ownership to the SYCL runtime. See section 4.4 for details. If
+the behavior is "transfer" then the runtime is going to destroy the input
+Level-Zero kernel</td>
+</tr><tr>
+<td>
+
+``` C++
+make_kernel<backend::ext_oneapi_level_zero>(
+    const backend_input_t<
+        backend::ext_oneapi_level_zero, kernel> &,
+    const context &Context)
+```
+</td>

```

Comment:
It looks like the descriptions for `make_kernel` and `make_kernel_bundle` are reversed.  (This is easier to see in the HTML render.)

It would also be nice if the order in this table matched the order of the table in "Mapping of SYCL objects to Level-Zero handles".  That other table has `kernel_bundle` before `kernel`.  If you like that order, put `make_kernel_bundle` before `make_kernel` in this table.

Reply:
fixed